gulp.config.js

```diff
@@ -1,8 +1,11 @@
-var historyApiFallback = require('connect-history-api-fallback')
+if (global.env === 'dev') {
+    var historyApiFallback = require('connect-history-api-fallback');
+}
 
 module.exports = function () {
     var root = '';
     var src = root + 'src/';
+    var translations = src + 'i18n/*.json';
```

Comment:
Yes is not related. This is copy paste from working example that has localization added. If this problem related to node module separation will be solved. I can make another pull request and add localization and related unit tests.